### PR TITLE
Anas responsive mansory layout

### PR DESCRIPTION
**Pull request by anask601**

### CODE REVIEW

**Review by AlishaPrasad — APPROVED** [review]

File: tasks-by-users/anask601/responsive-mansory-layout/style.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+body
+{
+    background: #222;
+    color: #f0efeb;
+    display: flex;
+    min-width: 100vw;
+    
+    justify-content: center;
+    align-self: center;
```
Comment (faran4engg) [inline tasks-by-users/anask601/responsive-mansory-layout/style.css:21]: minor: display: flez belongs here.

**Review by faran4engg — APPROVED** [review]
left minor comments for future.... approving

File: tasks-by-users/anask601/responsive-mansory-layout/style.css (hunk 1 above)
Comment (anask601) [reply tasks-by-users/anask601/responsive-mansory-layout/style.css:21]: okay

File: tasks-by-users/anask601/responsive-mansory-layout/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+{
+    position: relative;
+    max-width: 100%;
+    display: grid;
+    margin: 40px;
+
+    grid-template-columns: repeat(auto-fill, minmax(300px,1fr));
+    grid-template-rows: minmax(100px, auto);
+    grid-auto-flow: dense;
+    grid-gap: 10px;
```
[the remark this answers is not in the document]
Comment (anask601) [reply tasks-by-users/anask601/responsive-mansory-layout/style.css]: @faran bhai i have added display: grid property

**Pull request merged**